ksqldb-execution/src/test/java/io/confluent/ksql/execution/plan/StreamSourceTest.java

```diff
@@ -46,26 +47,29 @@
   @Test
   public void shouldImplementEquals() {
     new EqualsTester()
-        .addEqualityGroup(
+        .addEqualityGroup (
+            new StreamSource(
+                properties1, "topic1", formats1, Optional.of(timestamp1), schema1, OptionalInt.of(0)),
             new StreamSource(
-                properties1, "topic1", formats1, Optional.of(timestamp1), schema1),
+                properties1, "topic1", formats1, Optional.of(timestamp1), schema1, OptionalInt.of(0)))
```

Comment:
`OptionalInt.of(0)` and `OptionalInt.empty()` are expected to compare the same, right? If so, it'd be nice to add a case for this into this unit test (and similarly for the other classes too).